src/app/window-handler.ts

```diff
@@ -514,6 +525,16 @@ export class WindowHandler {
             this.addWindow(opts.winKey, this.screenPickerWindow);
         });
         ipcMain.once('screen-source-selected', (_event, source) => {
+            if (isWindowsOS) {
+                const str = JSON.stringify(source);
+                logger.info(`window-handler: screen-source-selected, source: ${str} id: ${id}!`);
```

Comment:
logger's internal method stringify objects when passed as an argument
```suggestion
                logger.info(`window-handler: screen-source-selected`, source, id);
```

Reply:
ok, cool, I'll change it